src/components/NamespaceList/__tests__/NamespaceListRow.spec.tsx

```diff
@@ -64,6 +64,19 @@ describe('NamespaceListRow', () => {
 
     expect(screen.getByText('Go to the namespace')).toBeInTheDocument();
   });
+
+  it('should make the namespace name a clickable link to the same target as the button', () => {
+    (useApplications as jest.Mock).mockReturnValue([[], true]);
+
+    routerRenderer(<NamespaceListRow columns={[]} obj={mockNamespace} />);
+
+    // Debug: log the DOM output
```

Comment:
for the debug, it is quite useful during the development. 
but we do not recommend to leave the debug before review & merge.
so how about remove the debug?